src/js/components/awardv2/idv/TablesSection.jsx

```diff
@@ -0,0 +1,111 @@
+/**
+ * TablesSection.jsx
+ * Created by David Trinh 12/10/18
+ **/
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { concat } from 'lodash';
+
+import TransactionsTableContainer from 'containers/awardV2/table/TransactionsTableContainer';
+
+import DetailsTabBar from '../../award/details/DetailsTabBar';
+import ResultsTablePicker from '../../search/table/ResultsTablePicker';
+
+const propTypes = {
+    overview: PropTypes.object,
+    activeTab: PropTypes.string,
+    clickTab: PropTypes.func
+};
+
+const commonTabs = [
+    {
+        label: 'Transaction History',
+        internal: 'transaction',
+        enabled: true
+    },
+    {
```

Comment:
Could you remove the Federal Account funding tab until we implement that feature?